WMCDuplicateRemover/Code/Serialization/ObjectDeserializer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace WMCDuplicateRemover.Code.Serialization
+{
+    public class ObjectDeserializer
+    {
+        public T ReadFromBinaryFile<T>(string filePath)
+        {
+            try
+            {
+                using (var stream = File.Open(filePath, FileMode.Open))
```

Comment:
Well, I was close.  Thanks for clearing that up.  I know we talked about it on trampoline day, I just wanted to know if there was a reason beyond the convenience of not naming a specific (and potentially very long) data type.